llvm/utils/TableGen/AsmMatcherEmitter.cpp

```diff
@@ -1712,11 +1712,11 @@ void AsmMatcherInfo::buildInstructionOperandReference(MatchableInfo *II,
   MatchableInfo::AsmOperand *Op = &II->AsmOperands[AsmOpIdx];
 
   // Map this token to an operand.
-  unsigned Idx;
-  if (!Operands.hasOperandNamed(OperandName, Idx))
+  std::optional<unsigned> MayBeIdx = Operands.findOperandNamed(OperandName);
```

Comment:
MayBe->Maybe. But I'd probably just name it Idx and use *Idx throughout?

Reply:
Done.